x/tally/keeper/filter.go

```diff
@@ -16,10 +16,53 @@ const (
 )
 
 type FilterResult struct {
-	Outliers     []int    // outlier list
-	Consensus    bool     // whether consensus was reached
-	ProxyPubKeys []string // consensus data proxy public keys
-	GasUsed      uint64   // gas used for filter
+	Errors       []bool   // i-th item is true if i-th reveal is non-zero exit or corrupt
+	Outliers     []int    // i-th item is non-zero if i-th reveal is an outlier
```

Comment:
Maybe it makes sense to have it as a `[]bool` here as well and only convert it to a `[]int` when we pass it to the tallyvm? Feels a bit odd to have that implementation detail leak into the filters here.

Reply:
Addressed in [bc1f126](https://github.com/sedaprotocol/seda-chain/pull/452/commits/bc1f12662c403d401343f9e894e487768d8ddc55). Actually, tally vm execution testing passed even though I didn't convert it to integers when passing it as an argument. Would it be ok to just pass the outlier list argument as JSON array of booleans?

Reply:
Where are these outliers being passed to? the tallyvm? Or somewhere else?

Reply:
Outlier list is one of the arguments. It is now a JSON array of booleans rather than of ints. I forget why I used ints initially, but I think it should be up to the wasm program to parse the outlier list correctly.